### PR TITLE
New language independent course

### PR DESCRIPTION
**Pull request by SynneK1337**
## What does this PR do?
Add resource(s)

## For resources
Course for writing operating system from scratch

### Why is this valuable (or not)?
It shows how to write operating system from absolute scratch
### How do we know it's really free?
You can just watch it

### For book lists, is it a book? For course lists, is it a course? etc.
It's a course
## Checklist:
- [*] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [*] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [*] Include author(s) and platform where appropriate.
- [*] Put lists in alphabetical order, correct spacing.
- [*] Add needed indications (PDF, access notes, under construction).
- [*] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-pl.md [hunk 1]
```diff
@@ -15,7 +16,8 @@
 * [Python](#python)
 * [Swift](#swift)
 
-
+### 0 - Niezależne od języka programowania
+* [OSDev](#0---niezale%C5%BCne-od-j%C4%99zyka-programowania) - Gynvael Coldwind
```
Comment (eshellman) [inline courses/free-courses-pl.md:20]: not a link

Comment (SynneK1337) [reply courses/free-courses-pl.md]: Fixed it

**Comment by SynneK1337** [comment]
Fixed it

File: courses/free-courses-pl.md [hunk 2]
```diff
@@ -15,7 +16,8 @@
 * [Python](#python)
 * [Swift](#swift)
 
-
+### 0 - Niezależne od języka programowania
+* [OSDev](https://www.youtube.com/playlist?list=PLGhEqiD7wAd5D-fe-Qz027_1yAH3rFMYF) - Gynvael Coldwind
```
Comment (shrn01) [inline courses/free-courses-pl.md:20]:
```suggestion
* [OSDev](https://www.youtube.com/playlist?list=PLGhEqiD7wAd5D-fe-Qz027_1yAH3rFMYF) - Gynvael Coldwind


```

File: courses/free-courses-pl.md [hunk 3]
```diff
@@ -15,6 +16,9 @@
 * [Python](#python)
 * [Swift](#swift)
```
Comment (shrn01) [inline courses/free-courses-pl.md:18]:
```suggestion


```

File: courses/free-courses-pl.md [hunk 4]
```diff
@@ -15,6 +16,9 @@
...
 
+### 0 - Niezależne od języka programowania
```
Comment (shrn01) [inline courses/free-courses-pl.md:19]:
```suggestion
### 0 - Niezależne od języka programowania

```

**Pull request merged**